bids/variables/variables.py

```diff
@@ -45,7 +45,7 @@ def clone(self, data=None, **kwargs):
         """
         result = deepcopy(self)
         if data is not None:
-            if data.shape != self.values.shape:
+            if data.squeeze().shape != self.values.squeeze().shape:
```

Comment:
Do you want to permit clones with different shapes to come out? Or are you just trying to loosen the restriction to ensure that a reshaping doesn't reorder the data?

```suggestion
            if data.squeeze().shape == self.values.squeeze().shape:
                data = data.reshape(self.values.shape)
            else:
```

Reply:
What was happening is one variable was say (1000,) while another was (1000,1). Squeeze allowed this comparison to pass, which will result in a valid output.